zilencer/management/commands/populate_billing_realms.py

```diff
@@ -0,0 +1,212 @@
+import contextlib
+from dataclasses import dataclass
+from typing import Any
+
+import stripe
+from django.core.management.base import BaseCommand
+from django.utils.timezone import now as timezone_now
+from typing_extensions import override
+
+from corporate.lib.stripe import update_or_create_stripe_customer
+from corporate.models import Customer, CustomerPlan, LicenseLedger
+from zerver.actions.create_realm import do_create_realm
+from zerver.actions.create_user import do_create_user
+from zerver.actions.streams import bulk_add_subscriptions
+from zerver.apps import flush_cache
+from zerver.lib.streams import create_stream_if_needed
+from zerver.models import Realm, UserProfile, get_realm
+from zproject.config import get_secret
+
+
+@dataclass
+class CustomerProfile:
+    unique_id: str
+    billing_schedule: int = CustomerPlan.ANNUAL
+    tier: int = CustomerPlan.STANDARD
+    automanage_licenses: bool = False
+    status: int = CustomerPlan.ACTIVE
+    sponsorship_pending: bool = False
+    card: str = ""
+
+
+class Command(BaseCommand):
+    help = "Populate database with different types of realms that can exist."
+
+    @override
+    def handle(self, *args: Any, **options: Any) -> None:
+        # Create a realm for each plan type
+        for plan_type, plan_name in Realm.ALL_PLAN_TYPES:
+            if plan_type == Realm.PLAN_TYPE_SELF_HOSTED:
+                continue
```

Comment:
This outer loop doesn't make sense to me; the `--annual-standard` plan should only be possible if the plan type is STANDARD, not LIMITED or PLUS.